### PR TITLE
Adds Background (first scss based) story to storybook

### PR DESCRIPTION
**Pull request by nikola-nulogy**
# Why
Adds Background story storybook. This story is the first .scss based page.

# What I Did
1. Created Bgrs.scss and Bgrs.story.js files in Nikola's sandbox
2. Transfered content of the files that I worked locally to newly created files

# How to Test
run $ yarn hack and go to Backgrounds story in Nikola's sandbox

### CODE REVIEW

**Review by theinterned — COMMENTED** [review]
I left a few sass-style issues. Perhaps we need a linter?

More importantly though, it looks like @matthewlyle 's work on type scale has crept into the PR?

File: sandbox/src/Nikola/Bgrs.scss [hunk 1]
```diff
@@ -0,0 +1,884 @@
+@import 'Bgrs/reset';
+@import 'Bgrs/variables';
+@import 'Bgrs/typography';
+@import 'Bgrs/utilities';
+
+.bgrs{
+  .nav-bar{
```
[the remark this answers is not in the document]
Comment (nikola-nulogy) [reply sandbox/src/Nikola/Bgrs.scss:7]: I needed this when I worked locally to get around some things. This needs additional clean up.

File: sandbox/src/Nikola/Bgrs.scss [hunk 2]
```diff
@@ -0,0 +1,884 @@
...
+
+.button{
+  &__primary{
+    background: $blue-dark;
+    border: solid 1px $blue-dark;
+    color: $white;
+    padding: 4px;
+    border-radius: 2px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (nikola-nulogy) [reply sandbox/src/Nikola/Bgrs.scss:922]: Also still in progress.

[the remark this answers is not in the document]
Comment (theinterned) [reply sandbox/src/Nikola/Bgrs.scss:922]: 😉

**Comment by theinterned** [comment]
What is the status of this PR @nikola-nulogy ? If it's no longer relevant, feel free to close it. Thanks!

**Pull request closed without merging**